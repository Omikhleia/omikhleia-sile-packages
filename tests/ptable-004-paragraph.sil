\begin{document}
\script[src=packages/ptable]

% Table containing paragraphs and long text that will be line-broken:
% Let us not forget these cells are parboxes.
% (This table was in a W3C documentation.)

\set[parameter=document.parindent, value=1em]

\begin[cols=26%fw 21%fw 37%fw 8%fw 8%fw, header=true]{ptable}
  \begin[background=#eee]{row}
    \celltable[span=5]{
      \row{\cell[span=5]{\center{Community Courses - Bath Autumn 1997}}}
      \begin{row}
        \cell{\center{Course Name}}
        \cell{\center{Course Tutor}}
        \cell{\center{Summary}}
        \cell{\center{Code}}
        \cell{\center{Fee}}
      \end{row}
    }
  \end{row}
  \begin{row}
    \cell{\noindent\raggedright{After the Civil War}}
    \cell{Dr. John Wroughton}
    \cell{\noindent\raggedright{%
      The course will examine the turbulent years in England
      after 1646.}
            \raggedleft{\em{6 weekly meetings starting Monday 13th
      October.}}}
    \cell{H27}
    \cell{£32}
  \end{row}
  \begin{row}
    \cell{\noindent\raggedright{An Introduction to Anglo-Saxon England}}
    \cell{Mark Cottle}
    \cell{\noindent\raggedright{%
      One day course introducing the early medieval
      period reconstruction the Anglo-Saxons and
      their society.}
      \raggedleft{\em{Saturday 18th October.}}}
    \cell{H28}
    \cell{£18}
  \end{row}
  \begin{row}
    \cell{\noindent\raggedright{The Glory that was Greece}}
    \cell{Valerie Lorenz}
    \cell{\noindent\raggedright{%
      Birthplace of democracy, philosophy, heartland of theater, home of
      argument.\par
      
      The Romans may have done it but the Greeks did it
      first.}
        \raggedleft{\em{Saturday day school 25th October.}}}
    \cell{H30}
    \cell{£18}
  \end{row}
\end{ptable}

\end{document}