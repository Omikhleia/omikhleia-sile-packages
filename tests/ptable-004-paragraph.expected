Set paper size 	595.275597	841.8897729
Begin page
Push color	0.9333	0.9333	0.9333
Draw line	29.7638	42.0945	535.7480	37.1914
Pop color
Mx 	29.7638
My 	60.6902
My 	51.3923
Draw line	29.7638	42.0945	535.7480	0.4000
Draw line	29.7638	60.6902	535.7480	0.4000
Draw line	29.7638	42.0945	0.4000	18.9957
Draw line	565.5118	42.0945	0.4000	18.9957
Mx 	213.4947
My 	54.2488
Set font 	Gentium Plus;10;400;;normal;;LTR
T	38 82 80 80 88 81 76 87 92 w=48.3545	(Community)
Mx 	264.4918
T	38 82 88 85 86 72 86 w=31.9922	(Courses)
Mx 	299.1266
T	16 w=3.3691	(-)
Mx 	305.1383
T	37 68 87 75 w=19.0479	(Bath)
Mx 	326.8287
T	36 88 87 88 80 81 w=33.5400	(Autumn)
Mx 	363.0113
T	20 28 28 26 w=18.7695	(1997)
Mx 	29.7638
My 	69.9880
Draw line	29.7638	60.6902	139.2945	0.4000
Draw line	29.7638	79.2859	139.2945	0.4000
Draw line	29.7638	60.6902	0.4000	18.9957
Draw line	169.0583	60.6902	0.4000	18.9957
Mx 	72.1596
My 	72.8445
T	38 82 88 85 86 72 w=28.1299	(Course)
Mx 	102.9320
T	49 68 80 72 w=23.7305	(Name)
Draw line	169.0583	60.6902	112.5071	0.4000
Draw line	169.0583	79.2859	112.5071	0.4000
Draw line	169.0583	60.6902	0.4000	18.9957
Draw line	281.5654	60.6902	0.4000	18.9957
Mx 	198.3069
T	38 82 88 85 86 72 w=28.1299	(Course)
Mx 	229.0794
T	55 88 87 82 85 w=23.2373	(Tutor)
Draw line	281.5654	60.6902	198.2268	0.4000
Draw line	281.5654	79.2859	198.2268	0.4000
Draw line	281.5654	60.6902	0.4000	18.9957
Draw line	479.7921	60.6902	0.4000	18.9957
Mx 	360.8765
T	54 88 80 80 68 85 92 w=39.6045	(Summary)
Draw line	479.7921	60.6902	42.8598	0.4000
Draw line	479.7921	79.2859	42.8598	0.4000
Draw line	479.7921	60.6902	0.4000	18.9957
Draw line	522.6520	60.6902	0.4000	18.9957
Mx 	491.1171
T	38 82 71 72 w=20.2100	(Code)
Draw line	522.6520	60.6902	42.8598	0.4000
Draw line	522.6520	79.2859	42.8598	0.4000
Draw line	522.6520	60.6902	0.4000	18.9957
Draw line	565.5118	60.6902	0.4000	18.9957
Mx 	537.0726
T	41 72 72 w=14.0186	(Fee)
Mx 	29.7638
My 	100.5837
Draw line	29.7638	79.2859	139.2945	0.4000
Draw line	29.7638	121.8816	139.2945	0.4000
Draw line	29.7638	79.2859	0.4000	42.9957
Draw line	169.0583	79.2859	0.4000	42.9957
Mx 	33.7638
My 	103.4402
T	36 73 87 72 85 w=21.0840	(After)
Mx 	57.4903
T	87 75 72 w=13.5791	(the)
Mx 	73.7120
T	38 76 89 76 79 w=18.4229	(Civil)
Mx 	94.7775
T	58 68 85 w=17.0508	(War)
Draw line	169.0583	79.2859	112.5071	0.4000
Draw line	169.0583	121.8816	112.5071	0.4000
Draw line	169.0583	79.2859	0.4000	42.9957
Draw line	281.5654	79.2859	0.4000	42.9957
Mx 	183.0583
T	39 85 w=9.8877	(Dr)
Mx 	192.9460
T	17 w=2.2900	(.)
Mx 	197.8798
T	45 82 75 81 w=19.1162	(John)
Mx 	219.6397
T	58 85 82 88 74 75 87 82 81 w=47.1729	(Wroughton)
Draw line	281.5654	79.2859	198.2268	0.4000
Draw line	281.5654	121.8816	198.2268	0.4000
Draw line	281.5654	79.2859	0.4000	42.9957
Draw line	479.7921	79.2859	0.4000	42.9957
Mx 	285.5654
My 	91.4402
T	55 75 72 w=15.6445	(The)
Mx 	303.8525
T	70 82 88 85 86 72 w=27.1387	(course)
Mx 	333.6337
T	90 76 79 79 w=15.1904	(will)
Mx 	351.4667
T	72 91 68 80 76 81 72 w=35.1172	(examine)
Mx 	389.2265
T	87 75 72 w=13.5791	(the)
Mx 	405.4482
T	87 88 85 69 88 79 72 81 87 w=39.3652	(turbulent)
Mx 	447.4560
T	92 72 68 85 86 w=21.9629	(years)
Mx 	285.5654
My 	103.4402
T	76 81 w=8.2275	(in)
Mx 	296.4355
T	40 81 74 79 68 81 71 w=33.3740	(England)
Mx 	332.4521
T	68 73 87 72 85 w=19.7217	(after)
Mx 	354.8163
T	20 25 23 25 w=18.7695	(1646)
Mx 	373.5859
T	17 w=2.2900	(.)
Mx 	291.8419
My 	115.4402
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	25 w=4.0186	(6)
Mx 	298.2628
T	90 72 72 78 79 92 w=25.4199	(weekly)
Mx 	326.0851
T	80 72 72 87 76 81 74 86 w=33.4668	(meetings)
Mx 	361.9542
T	86 87 68 85 87 76 81 74 w=29.5166	(starting)
Mx 	393.8732
T	48 82 81 71 68 92 w=30.2930	(Monday)
Mx 	426.5685
T	20 22 87 75 w=16.0498	(13th)
Mx 	445.0206
T	50 70 87 82 69 72 85 w=28.7695	(October)
Mx 	473.7902
T	17 w=2.0020	(.)
Draw line	479.7921	79.2859	42.8598	0.4000
Draw line	479.7921	121.8816	42.8598	0.4000
Draw line	479.7921	79.2859	0.4000	42.9957
Draw line	522.6520	79.2859	0.4000	42.9957
Mx 	493.7921
My 	103.4402
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 21 26 w=15.9229	(H27)
Draw line	522.6520	79.2859	42.8598	0.4000
Draw line	522.6520	121.8816	42.8598	0.4000
Draw line	522.6520	79.2859	0.4000	42.9957
Draw line	565.5118	79.2859	0.4000	42.9957
Mx 	536.6520
T	133 w=4.6924	(£)
Mx 	541.3444
T	22 21 w=9.3848	(32)
Mx 	29.7638
My 	149.1794
Draw line	29.7638	121.8816	139.2945	0.4000
Draw line	29.7638	176.4773	139.2945	0.4000
Draw line	29.7638	121.8816	0.4000	54.9957
Draw line	169.0583	121.8816	0.4000	54.9957
Mx 	33.7638
My 	146.0359
T	36 81 w=11.4697	(An)
Mx 	47.8761
T	44 81 87 85 82 71 88 70 87 76 82 81 w=52.4951	(Introduction)
Mx 	103.0138
T	87 82 w=8.4717	(to)
Mx 	114.1280
T	36 81 74 79 82 w=24.0869	(Anglo)
Mx 	138.2150
T	16 w=3.3691	(-)
Mx 	33.7638
My 	158.0359
T	54 68 91 82 81 w=24.9268	(Saxon)
Mx 	61.3331
T	40 81 74 79 68 81 71 w=33.3740	(England)
Draw line	169.0583	121.8816	112.5071	0.4000
Draw line	169.0583	176.4773	112.5071	0.4000
Draw line	169.0583	121.8816	0.4000	54.9957
Draw line	281.5654	121.8816	0.4000	54.9957
Mx 	183.0583
My 	152.0359
T	48 68 85 78 w=21.4893	(Mark)
Mx 	207.1951
T	38 82 87 87 79 72 w=24.6045	(Cottle)
Draw line	281.5654	121.8816	198.2268	0.4000
Draw line	281.5654	176.4773	198.2268	0.4000
Draw line	281.5654	121.8816	0.4000	54.9957
Draw line	479.7921	121.8816	0.4000	54.9957
Mx 	285.5654
My 	134.0359
T	50 81 72 w=16.1377	(One)
Mx 	304.3456
T	71 68 92 w=14.7217	(day)
Mx 	321.7099
T	70 82 88 85 86 72 w=27.1387	(course)
Mx 	351.4911
T	76 81 87 85 82 71 88 70 76 81 74 w=48.6328	(introducing)
Mx 	402.7665
T	87 75 72 w=13.5791	(the)
Mx 	418.9882
T	72 68 85 79 92 w=20.8105	(early)
Mx 	285.5654
My 	146.0359
T	80 72 71 76 72 89 68 79 w=37.4121	(medieval)
Mx 	325.6200
T	83 72 85 76 82 71 w=26.8164	(period)
Mx 	355.0790
T	85 72 70 82 81 86 87 85 88 70 87 76 82 81 w=61.1279	(reconstruction)
Mx 	418.8495
T	87 75 72 w=13.5791	(the)
Mx 	435.0712
T	36 81 74 79 82 w=24.0869	(Anglo)
Mx 	459.1581
T	16 w=3.3691	(-)
Mx 	285.5654
My 	158.0359
T	54 68 91 82 81 86 w=28.7891	(Saxons)
Mx 	316.9970
T	68 81 71 w=15.3076	(and)
Mx 	334.9472
T	87 75 72 76 85 w=20.2490	(their)
Mx 	357.8388
T	86 82 70 76 72 87 92 w=28.9648	(society)
Mx 	386.8036
T	17 w=2.2900	(.)
Mx 	389.9083
My 	170.0359
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	54 68 87 88 85 71 68 92 w=34.2578	(Saturday)
Mx 	426.5685
T	20 27 87 75 w=16.0498	(18th)
Mx 	445.0206
T	50 70 87 82 69 72 85 w=28.7695	(October)
Mx 	473.7902
T	17 w=2.0020	(.)
Draw line	479.7921	121.8816	42.8598	0.4000
Draw line	479.7921	176.4773	42.8598	0.4000
Draw line	479.7921	121.8816	0.4000	54.9957
Draw line	522.6520	121.8816	0.4000	54.9957
Mx 	493.7921
My 	152.0359
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 21 27 w=15.9229	(H28)
Draw line	522.6520	121.8816	42.8598	0.4000
Draw line	522.6520	176.4773	42.8598	0.4000
Draw line	522.6520	121.8816	0.4000	54.9957
Draw line	565.5118	121.8816	0.4000	54.9957
Mx 	536.6520
T	133 w=4.6924	(£)
Mx 	541.3444
T	20 27 w=9.3848	(18)
Mx 	29.7638
My 	209.7752
Draw line	29.7638	176.4773	139.2945	0.4000
Draw line	29.7638	243.0730	139.2945	0.4000
Draw line	29.7638	176.4773	0.4000	66.9957
Draw line	169.0583	176.4773	0.4000	66.9957
Mx 	33.7638
My 	212.6316
T	55 75 72 w=15.6445	(The)
Mx 	52.0509
T	42 79 82 85 92 w=22.4609	(Glory)
Mx 	77.1544
T	87 75 68 87 w=16.9922	(that)
Mx 	96.7892
T	90 68 86 w=15.5127	(was)
Mx 	114.9444
T	42 85 72 72 70 72 w=28.0176	(Greece)
Draw line	169.0583	176.4773	112.5071	0.4000
Draw line	169.0583	243.0730	112.5071	0.4000
Draw line	169.0583	176.4773	0.4000	66.9957
Draw line	281.5654	176.4773	0.4000	66.9957
Mx 	183.0583
T	57 68 79 72 85 76 72 w=29.7070	(Valerie)
Mx 	215.4116
T	47 82 85 72 81 93 w=28.1299	(Lorenz)
Draw line	281.5654	176.4773	198.2268	0.4000
Draw line	281.5654	243.0730	198.2268	0.4000
Draw line	281.5654	176.4773	0.4000	66.9957
Draw line	479.7921	176.4773	0.4000	66.9957
Mx 	285.5654
My 	188.6316
T	37 76 85 87 75 83 79 68 70 72 w=42.7148	(Birthplace)
Mx 	330.9228
T	82 73 w=8.1396	(of)
Mx 	341.7050
T	71 72 80 82 70 85 68 70 92 w=45.1025	(democracy)
Mx 	386.8075
T	15 w=2.2900	(,)
Mx 	391.7402
T	83 75 76 79 82 86 82 83 75 92 w=45.9033	(philosophy)
Mx 	437.6435
T	15 w=2.2900	(,)
Mx 	285.5654
My 	200.6316
T	75 72 68 85 87 79 68 81 71 w=40.1465	(heartland)
Mx 	328.3544
T	82 73 w=8.1396	(of)
Mx 	339.1366
T	87 75 72 68 87 72 85 w=30.1904	(theater)
Mx 	369.3271
T	15 w=2.2900	(,)
Mx 	374.2597
T	75 82 80 72 w=23.1982	(home)
Mx 	400.1005
T	82 73 w=8.1396	(of)
Mx 	410.8827
T	68 85 74 88 80 72 81 87 w=40.3369	(argument)
Mx 	451.2197
T	17 w=2.2900	(.)
Mx 	285.5654
My 	212.6316
T	55 75 72 w=15.6445	(The)
Mx 	303.8525
T	53 82 80 68 81 86 w=32.6416	(Romans)
Mx 	339.1366
T	80 68 92 w=17.5537	(may)
Mx 	359.3329
T	75 68 89 72 w=19.6582	(have)
Mx 	381.6337
T	71 82 81 72 w=20.3662	(done)
Mx 	404.6425
T	76 87 w=6.1523	(it)
Mx 	413.4374
T	69 88 87 w=13.8184	(but)
Mx 	429.8984
T	87 75 72 w=13.5791	(the)
Mx 	446.1200
T	42 85 72 72 78 86 w=27.7686	(Greeks)
Mx 	285.5654
My 	224.6316
T	71 76 71 w=13.1104	(did)
Mx 	301.3183
T	76 87 w=6.1523	(it)
Mx 	310.1132
T	737 85 86 87 w=17.0850	(first)
Mx 	327.1982
T	17 w=2.2900	(.)
Mx 	348.2677
My 	236.6316
Set font 	Gentium Plus;10;400;Italic;normal;;LTR
T	54 68 87 88 85 71 68 92 w=34.2578	(Saturday)
Mx 	384.9279
T	71 68 92 w=13.7549	(day)
Mx 	401.0851
T	86 70 75 82 82 79 w=23.0811	(school)
Mx 	426.5685
T	21 24 87 75 w=16.0498	(25th)
Mx 	445.0206
T	50 70 87 82 69 72 85 w=28.7695	(October)
Mx 	473.7902
T	17 w=2.0020	(.)
Draw line	479.7921	176.4773	42.8598	0.4000
Draw line	479.7921	243.0730	42.8598	0.4000
Draw line	479.7921	176.4773	0.4000	66.9957
Draw line	522.6520	176.4773	0.4000	66.9957
Mx 	493.7921
My 	212.6316
Set font 	Gentium Plus;10;400;;normal;;LTR
T	43 22 19 w=15.9229	(H30)
Draw line	522.6520	176.4773	42.8598	0.4000
Draw line	522.6520	243.0730	42.8598	0.4000
Draw line	522.6520	176.4773	0.4000	66.9957
Draw line	565.5118	176.4773	0.4000	66.9957
Mx 	536.6520
T	133 w=4.6924	(£)
Mx 	541.3444
T	20 27 w=9.3848	(18)
End page
Finish
